python-sdk/mk/local.mk

```
@@ -37,6 +37,7 @@ install: virtualenv  ## Install python dependencies in existing virtualenv
 	@$(PIP) install pre-commit
 	@cd .. && $(PIP) install -e .[all]
 	@cd .. && $(PIP) install .[tests]
+	@cd .. && $(PIP) install .[doc]
```

Should we just change this to the following:

```suggestion
	@cd .. && $(PIP) install -e .[all, tests, doc]
```